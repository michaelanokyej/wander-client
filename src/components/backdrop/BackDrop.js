import React from "react";
import "./BackDrop.css";
import toursAndUserContext from "../context/toursAndUserContext.js";

class BackDrop extends React.Component {
  static contextType = toursAndUserContext;

  render() {
    return (
      <div
        className="backdrop"
        onClick={e => this.context.backDropClickHandler()}
      />
    );
  }
}

export default BackDrop;
