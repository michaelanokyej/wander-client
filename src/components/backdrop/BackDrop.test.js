import React from "react";
import BackDrop from "./BackDrop"
import { shallow } from "enzyme"
import toJson from "enzyme-to-json"
import renderer from 'react-test-renderer'

describe(`BackDrop component`, () => {
    it(`renders the BackDrop feature`, () => {
        const wrapper = shallow(<BackDrop />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});