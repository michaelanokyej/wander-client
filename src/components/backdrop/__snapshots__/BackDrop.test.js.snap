// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackDrop component renders the BackDrop feature 1`] = `
<div
  className="backdrop"
  onClick={[Function]}
/>
`;
